canonical-web-and-design/ubuntu.com
# Fix blog article 500s

This fixes the template rendering race-condition in extends_with_args
(see 8de572e8 for details).

I'm also re-enabling the blog by removing the `/blog` redirect.

QA
--

`./run --env DJANGO_DEBUG=false`

Then go to a blog article, e.g. http://0.0.0.0:8001/blog/canonical-announces-support-for-ubuntu-on-windows-subsystem-for-linux-2, and reload it at least 20 times. Check you don't see any errors, missing content, missing titles etc.
